src/Nethermind/Nethermind.JsonRpc.Test/Modules/Eth/EthRpcModuleTests.cs

```diff
@@ -41,7 +41,6 @@ namespace Nethermind.JsonRpc.Test.Modules.Eth;
 
 [Parallelizable(ParallelScope.All)]
 [TestFixture]
-[Culture("en-US")]
```

Comment:
Why disable culture? Test fail on different culture without it

Reply:
This attribute causes many tests to be skipped by the test runner for some reason. 
https://github.com/NethermindEth/nethermind/actions/runs/8394059990/job/22990371946#step:4:77

The solution is to give `CultureInfo.Invariant` in the json converters instead.

Reply:
wait why?

Reply:
https://docs.nunit.org/articles/nunit/writing-tests/attributes/culture.html

Aha got it, probably tests on CI use different culture to run.

It should be `[SetCulture]` attrubute - can you fix in all places in our codebase?

Reply:
Other places are fine